package.json

```diff
@@ -12,23 +12,29 @@
     "manifest:chrome": "BROWSER=CHROME ts-node ./scripts/create-manifest.ts",
     "manifest:edge": "BROWSER=EDGE ts-node ./scripts/create-manifest.ts",
     "manifest:dev": "NODE_ENV=development npm run manifest:chrome",
-    "build:firefox": "BROWSER=FIREFOX npm run manifest:firefox; BROWSER=FIREFOX npm run script:make-zip",
-    "build:chrome": "BROWSER=CHROME npm run manifest:chrome; BROWSER=CHROME npm run script:make-zip",
-    "build:edge": "BROWSER=EDGE npm run manifest:edge; BROWSER=EDGE npm run script:make-zip",
+    "build:firefox": "BROWSER=FIREFOX run-s manifest:firefox script:make-zip",
+    "build:chrome": "BROWSER=CHROME run-s manifest:chrome script:make-zip",
+    "build:edge": "BROWSER=EDGE run-s manifest:edge script:make-zip",
     "build:dev": "NODE_ENV=development; webpack -w --mode development",
-    "build:prod": "NODE_ENV=production; webpack --mode production --config webpack.config.js; npm run clean; npm run build:firefox; npm run build:chrome; npm run build:edge",
+    "build:prod": "NODE_ENV=production; webpack --mode production --config webpack.config.js; run-s clean build:firefox build:chrome build:edge",
     "dist": "npm run build:prod",
-    "lint": "npm run lint:packages && npm run lint:json && npm run lint:typings",
-    "lint:packages": "prettier --write ./packages/**/*.{ts,d.ts,tsx}",
-    "lint:json": "prettier --parser json --write ./*.json",
-    "lint:typings": "prettier --write ./typings/**/*.d.ts",
+    "lint": "run-s lint:packages lint:json lint:typings",
+    "lint:packages": "prettier --check ./packages/**/*.{ts,d.ts,tsx}",
+    "lint:json": "prettier --parser json --check ./{*,tests/*}.json",
+    "lint:typings": "prettier --check ./typings/**/*.d.ts",
+    "format": "run-s format:packages format:json format:typings",
+    "format:packages": "prettier --write ./packages/**/*.{ts,d.ts,tsx}",
+    "format:json": "prettier --parser json --write ./{*,tests/*}.json",
+    "format:typings": "prettier --write ./typings/**/*.d.ts",
     "test": "jest --coverage"
   },
   "jest": {
```

Comment:
Maybe we could use config file for jest in near future. What do you think @s-weigand ?

Reply:
Sure, If you want I can move the config in this PR.

Reply:
That's ok, I'm going to merge it now.....